stripe/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -148,7 +148,7 @@ data class PaymentMethodCreateParams internal constructor(
                 Type.AuBecsDebit -> auBecsDebit?.toParamMap()
                 Type.BacsDebit -> bacsDebit?.toParamMap()
                 Type.Sofort -> sofort?.toParamMap()
-                Type.P24, Type.Bancontact, Type.Giropay, Type.Eps -> null
+                else -> null
```

Comment:
I kind of like being explicit to make sure we don't miss any (assuming that results in a "non-exhaustive match" warning) but don't think it's too much of a problem if we have good tests

Reply:
I think this will list get longer over time and make the class a bit less readable. I expect that we'll write tests for new types so I think this is fine.